hieroplatform/src/main/java/org/hiero/sketch/spreadsheet/Bucket1D.java

```diff
@@ -0,0 +1,84 @@
+package org.hiero.sketch.spreadsheet;
+
+import org.checkerframework.checker.nullness.qual.NonNull;
+
+/**
+ * A one dimensional bucket designed for a Histogram1D
+ */
+public class Bucket1D {
+    private Object minObject;
+    private Object maxObject;
+    private double minValue;
+    private double maxValue;
+    private long count;
+
+    public Bucket1D() { }
```

Comment:
The maxValue and minValue are a function of the items placed in the bucket, not the boundary of the bucket. There is no reason to construct one with them. There is a private construction function that gets all the fields and is used in the union operation.
Java by default initializes all objects to null/zero so it's just a style issue, I added initializations for readability.